toucan_connectors/snowflake/snowflake_connector.py

```diff
@@ -99,7 +99,18 @@ def get_connection_params(self):
             'authenticator': self.authentication_method,
         }
 
-        if self.authentication_method == AuthenticationMethod.PLAIN and self.password:
+        if not self.authentication_method:
+            # Default to User/Password authentication method if the parameter
+            # was not set when the connector was created
+            res['authenticator'] = AuthenticationMethod.PLAIN
+
+        # This test refers to the `authenticator` key value in `res` because
+        # the authentication method may not have been set at the creation of
+        # the connector.
+        # For now this is a way around to be able to default to plain
+        # user/password authentication when no authentication mechanism was
+        # specified by a conceptor.
+        if res['authenticator'] == AuthenticationMethod.PLAIN and self.password:
```

Comment:
Je pense que tu n’as pas besoin de mettre autant de commentaires . À la limite, juste : “the default authentication method is Plain”